chapters/hub/build-pod-image.md

```diff
@@ -1,39 +1,45 @@
 # Build Your Pod into a Docker Image
 
 ## Goal
+The objective of this tutorial is to serve as an elaborate guide for Pod image usage.
+Users can use Pod Image via several ways:
 
-Instead of 
-```bash
-jina pod --uses hub/example/mwu_encoder.yml --port-in 55555 --port-out 55556
-```
-
-After this tutorial, you can use the Pod image via:
-```bash
-docker run jinaai/hub.examples.mwu_encoder --port-in 55555 --port-out 55556
-```
-
-...or use the Pod image in the Flow API:
-```python
-from jina.flow import Flow
+- Run with Docker (`docker run`)
+  - ```
+    bash docker run jinaai/hub.examples.mwu_encoder --port-in 55555 --port-out 55556
+    ```
+    
+- Flow API
+  - ```python
+    from jina.flow import Flow
 
-f = (Flow()
+    f = (Flow()
         .add(name='my-encoder', image='jinaai/hub.examples.mwu_encoder', port_in=55555, port_out=55556)
         .add(name='my-indexer', uses='indexer.yml'))
-```
-
-... or use the Pod image via Jina CLI
-```bash
-jina pod --uses jinaai/hub.examples.mwu_encoder --port-in 55555 --port-out 55556
-```
-
+    ```
+    
+- Jina CLI
+  - ```
+    bash jina pod --uses jinaai/hub.examples.mwu_encoder --port-in 55555 --port-out 55556
+    ```
+    
+- Conventional local usage with `uses` flag
+  - ``` 
+    bash jina pod --uses hub/example/mwu_encoder.yml --port-in 55555 --port-out 55556
+    ```
+    
 More information about [the usage can be found here](./use-your-pod.html#use-your-pod-image).
 
 
 ## Why?
 
-So you have implemented an awesome executor and want to reuse it in another Jina application, or share it with people in the world. Kind as you are, you want to offer people a ready-to-use interface without hassling them to repeat all steps and pitfalls you have done. The best way is thus to pack everything (python file, YAML config, pre-trained data, dependencies) into a container image and use Jina as the entry point. You can also annotate your image with some meta information to facilitate the search, archive and classification.
+So you have implemented an awesome executor. You would want to reuse it in another Jina application or share it with people in the world. 
+
+Kind as you are, you want to offer people a ready-to-use interface without the hassle of repeating the pitfalls you faced.
 
-Here are a list of reasons that may motivate you to build a Pod image:
+The best way, thus, is to pack everything (`python` file, `YAML` config, `pre-trained` data, dependencies) into a container image and use Jina as the entry point. You can also annotate your image with some meta information to facilitate the search, archive and classification.
+
+Here is a list of motivating reasons for building a Pod image:
 
 - You want to use one of the built-in executor (e.g. pytorch-based) and you don't want to install pytorch dependencies on the host.
 - You modify or write a new executor and want to reuse it in another project, without touching [`jina-ai/jina`](https://github.com/jina-ai/jina/).
```

Comment:
**E**xecutor - please check all occurrences in this doc (don't auto-replace since we need lower-case in code fences)

Note: In this doc I'll skip comments for this capitalization issue going forwards

Reply:
without touching [Jina's core](github url...) (don't just use repo name)